pkg/vsphere/toolbox/service.go

```diff
@@ -223,6 +223,10 @@ func (s *Service) SetOption(args []byte) ([]byte, error) {
 	case "broadcastIP": // TODO: const-ify
 		if val == "1" {
 			ip := s.PrimaryIP()
+			if ip == "" {
+				log.Printf("failed to find primary IP\n")
```

Comment:
Right, vsphere/toolbox package currently uses stdlib log.. if we have a good reason to switch it to logsrus, let's do that in a separate PR that changes the rest of the logging too.